www/package.json

```diff
@@ -12,10 +12,11 @@
   },
   "scripts": {
     "analyze-bundle": "babel-node ./tools/analyzeBundle.js",
-    "build": "rimraf dist && yarn build:examples & yarn build:react && babel-node tools/build.js",
+    "build": "rimraf dist && yarn build:examples & yarn build:react & yarn build:fetch && babel-node tools/build.js",
     "build:watch": "yarn build && yarn open:dist",
     "build:examples": "rimraf ./client/examples && babel-node ./tools/copyExamples.js",
     "build:react": "cd ../react && yarn build:lib",
+    "build:fetch": "rimraf ./client/data && babel-node ./tools/fetchComponentList.js",
```

Comment:
I'm not sure if "fetch" is the right name for this since all of the data is in the repo. Fetch implies going and getting it from an api or somewhere externally to me. Could be confusing for future devs working on the system.